### PR TITLE
Bk customer surveys profile

### PR DESCRIPTION
**Pull request by kandelchy**

### CODE REVIEW

File: app/controllers/ads_controller.rb [hunk 1]
```diff
@@ -66,7 +66,7 @@ def map
   end
 
   def edit_map
-    @selected_prov = params[:prov].split(',')
+    @selected_prov = params[:prov]
```
Comment (ssharba) [inline app/controllers/ads_controller.rb]: @kandelchy did you made this change?

File: app/controllers/survey_forms_controller.rb [hunk 2]
```diff
@@ -18,13 +18,10 @@ def new
...
 
   def publish
-    @survey_request = SurveyRequest.where(survey_form_id: params[:id])
-    @survey_request.first.update(status: 4)
-
     @survey_form = SurveyForm.find(params[:id])
     @survey_form.is_published = true
     @survey_form.save
-    @survey_forms = SurveyForm.where(surveyable_type: 'User')
+    @survey_forms = SurveyForm.where(surveyable_type: 'Business', ad_id: nil)
```
Comment (ssharba) [inline app/controllers/survey_forms_controller.rb]: is ad_id optional? if you , make sure to add optional: true to the model

File: app/controllers/ads_controller.rb (hunk 1 above)
Comment (kandelchy) [reply app/controllers/ads_controller.rb]: Yes, there was an error on presentation day and I asked Mustafa and he give me this

File: app/controllers/survey_forms_controller.rb (hunk 2 above)
Comment (kandelchy) [reply app/controllers/survey_forms_controller.rb]: Ad has_one survey, so when the Survey is created without Ad then ad_id is blank

Comment (ssharba) [reply app/controllers/survey_forms_controller.rb]: add optional: true to SurveyForm for ad_id

Comment (kandelchy) [reply app/controllers/survey_forms_controller.rb]: OK

**Pull request merged**